package.json

```diff
@@ -0,0 +1,634 @@
+{
+  "name": "leaderboardproject",
+  "version": "1.0.0",
+  "description": "",
+  "private": "true",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Ruthmy/leaderboardProject.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Ruthmy/leaderboardProject/issues"
+  },
+  "homepage": "https://github.com/Ruthmy/leaderboardProject#readme",
```

Comment:
- [ ] Great job on implementing the setting up of your project as specified in the project requirement! 👏 However, there is a minor issue 🐞 in your `package.json` file that we need to address.
   - [ ] Kindly consider removing the highlighted sections from your `package.json` file as they are not necessary and due to that installing your project dependencies after cloning returns an `ERR`.
   - [ ] Also, Please remove the `dependencies` sections as well starting from the line `40` to `633`. Below is a screenshot of the `ERR` I received when these highlighted sections are in the file.
   
<img width="1082" alt="Screenshot 2023-06-22 at 00 10 57" src="https://github.com/Ruthmy/leaderboardProject/assets/107192076/3caa682d-f838-43b3-8194-bb860d67deb5">

Reply:
Hi @Strangeal ! Thanks for your feedback.
I have already removed the dependencies and the lines you requested to avoid the error.